### ISSUE TITLE
Apple Pencil double tap to switch between pen and eraser iPadOS

### ISSUE THREAD

**Issue by QuifixOfficial** [saber-notes/saber#1303]
### Before submitting an issue, please check the following

- [X] I have searched for similar issues (both open and closed) and cannot find a duplicate
- [X] I agree to follow the [Code of Conduct](https://github.com/saber-notes/saber/blob/main/CODE_OF_CONDUCT.md)

### Motivation

Double tapping on Apple Pencil second generation does not switch between the eraser and tool in use (e.g eraser)

### Solution

Implementing this feature 

### Saber version

v0.24.1 App Store (24010)

### Device

- Device:iPad air 4
- OS: latest iPadOS

**Comment by thad137** [comment]
iPad OS pencil gestures in general would be incredibly helpful. It’s the one thing that makes me miss other note apps.

**Comment by KalokaK** [comment]
Yes upvoting this. Quickly switching to eraser is one of the switcher keeping me on other software for now sadly.

**Comment by J-Bch** [comment]
Upvoting this as well. The app is really good, this is the only major improvement missing for my use case.

**Comment by Thatsean** [comment]
This is the only feature missing from the app. Upvoting as well.
